source/GovUK.Template.Razor.nuspec.erb

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>GovUK.Template.Razor</id>
+        <version><%= GovukTemplate::VERSION %></version>

```

Comment:
I think it's more explicit to set it in the package definition, so I'll remove the command line flag.